### DISCUSSION TITLE
[v5] seeder order

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#2159] [Help] [answered]
Hi there,

in which order will the seeder executed?  Is there a way to set a explicit order?

Bye

**Comment by ammezie** [comment, marked as answer]
You can prefix the seeders with numbers. Eg:

1-userSeeder.ts
2-postSeeder.ts
and so on..

**Reply by rbartholomay** [reply]
Oh! Nice..

This must be new for v5...  I remember that in v4 this will not correct run.

thanks!
